QueryBuilder/Clauses/ColumnClause.cs

```diff
@@ -59,6 +60,31 @@ public override AbstractClause Clone()
         }
     }
 
+    public class AggregateColumn : AbstractColumn
+    {
+        public string Type { get; set; } // Min, Max, etc.
+        public List<string> Columns { get; set; } // Count can do multiple (?)
```

Comment:
but the count with multiple doesn't use `AggregateColumn`? i.o.w all `AggregateColumn` uses only work with a single column so we should not have a List here.

Reply:
pushed an update